### PR TITLE
[v2.0.0]: PR for 2.0.0 release

### PR DESCRIPTION
**Pull request by jkuri**

### CODE REVIEW

**Comment by irmana** [comment]
This looks already like a great push @jkuri at a first glance. 

I've set the label "In Progress", feel free to change it to "Review Needed" once you're done. I am sure there will be more folks willing to provide a proper review once you think you're done from your side. I'll provide my review too.

**Comment by jkuri** [comment]
Thanks @irmana. Okay, I will also prepare online demo for this PR so it will be easier to check.

**Comment by belsakn** [comment]
Hey @jkuri ,
This look really good :+1: i check only the backend since i don't have much experience in frontend :)
Can't wait to see it in action.

**Comment by jkuri** [comment]
@belsakn I will try to set up demo on server asap. Till then you can check screen capture I just record to show how distributed builds on abstruse 2.0.0 will work, video is [here](https://www.youtube.com/watch?v=s4y_M1J-Tuw). Let me know what you think.

**Comment by irmana** [comment]
@jkuri that would be awesome... What are the major parts that still need to be completed? Also, it would be nice to indicate whether this is on frontend or backend.

File: server/api/builds/handlers.go [hunk 1]
```diff
@@ -12,17 +12,17 @@ import (
...
-	// }
-
-	// PR test
-	if err := core.StartBuild(11, 2, "master", "test pull request case for abstruse ci", "3a0cb5c3ad82caf3bb935032e85919dec273bdc8", "build(abstruse): abstruse test commit"); err != nil {
+	if err := core.StartBuild(1, 0, "abstruse-config-test", "", "3a1a22865810755508147331a08eb3af83946630", "chore(): update abstruse.yml configuration"); err != nil {
 		api.JSONResponse(res, http.StatusInternalServerError, api.ErrorResponse{Data: err.Error()})
 		return
 	}
 
+	// PR test
```
[the remark this answers is not in the document]
Comment (jkuri) [reply server/api/builds/handlers.go]: its a work in progress.

**Comment by belsakn** [comment]
@jkuri Looks like it works perfectly.

**Pull request closed without merging**